web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerDefault.java

```diff
@@ -0,0 +1,23 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+public class LinkCheckerDefault implements LinkCheckerInterface {
+    private String getCheckUrl(Element onlineResource) {
+        return LinkCheckerUtils.parseOnlineResource(onlineResource, LinkCheckerUtils.URL_XPATH);
+    }
+
+    @Override
+    public boolean check(Element onlineResource) {
```

Comment:
It's implemented slightly different in this case. I'm actually happy with this abstraction level:
 * One class for simple links
 * One class for WMS
 * One class for WFS

WMS and WFS are a bit similar, but for the sake of clarity and readability, I prefer to have those 5-10 duplicated code lines.